_mobile/home.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: mobile
+title: Home
+permalink: /mobile/home/
+background-class: mobile-background
+body-class: mobile
+order: 1
+published: true
+---
+
+# PyTorch Mobile
+
+Running ML on edge devices is growing in importance as applications continue to demand lower latency. It is also a foundational element for privacy-preserving techniques such as federated learning. As of PyTorch 1.3, PyTorch supports an end-to-end workflow from Python to deployment on iOS and Android. 
+
+This is an early, experimental release that we will be building on in several areas over the coming months:
+
+- Extends the PyTorch API to cover common preprocessing and integration tasks needed for incorporating ML in mobile applications
+- Support for FBGEMM and QNNPACK quantized kernel libraries support for x86 and ARM CPUs respectively
+- (Coming soon) Build level optimization and selective compilation depending on the operators needed for user applications (i.e., you pay binary size for only the operators you need)
```

Comment:
this entire list is Coming soon, so I'd drop it here

Reply:
+1 - let's remove for now..

Reply:
Yes, let's remove for now..  cc @brucejlin1

Reply:
Removed "Coming soon" text